docs/source/datasets.rst

```diff
@@ -244,7 +231,6 @@ a track.
 .. _medley_solos_db: https://mirdata.readthedocs.io/en/latest/source/mirdata.html#module-mirdata.medley_solos_db
 .. _medleydb_melody: https://mirdata.readthedocs.io/en/latest/source/mirdata.html#module-mirdata.medleydb_melody
 .. _medleydb_pitch: https://mirdata.readthedocs.io/en/latest/source/mirdata.html#module-mirdata.medleydb_pitch
-.. _mridangam_stroke: https://mirdata.readthedocs.io/en/latest/source/mirdata.html#module-mirdata.mridangam_stroke
```

Comment:
you will probably get a conflict on this line because mridangam_stroke is already in master. try to do a rebase ?

Reply:
Well, I just created a new branch from mirdata/master and added cante100's files there, so that I am working on top of the latest master version of the original mirdata repo. Shouldn't this be alright?

Reply:
just to check, fetch and merge with the latest origin/master
https://gist.github.com/whoisryosuke/36b3b41e738394170b9a7c230665e6b9#:~:text=Updating%20a%20feature%20branch&text=Merge%20the%20changes%20from%20origin,a%20"fast-forward".